src/actions/index.js

```diff
@@ -53,9 +53,7 @@ export const activeSearchToggle = () => {
   }
 }
 
-export const addFavorite = (SchoolData) => {
-  console.log('data in add fav ', SchoolData);
```

Comment:
I think this one was mine, but can we try to keep console logs off of master?  that would be awesome and save me some time going through and deleting console logs that clog up my console when trying to debug